### ISSUE TITLE
Automated testing

### ISSUE THREAD

**Issue by dashohoxha** [Own-Mailbox/omb#12]
It would be nice to have some automated testing, to make sure that the server works correctly after each change, and there are no regressions.
However I don't have much experience on this area, and I don't have any idea about how to do it.
Maybe @giorgos314 can spend some time exploring this, even if he does not succeed.

**Comment by giorgos314** [comment]
Yes, of course. I don't have much experience either but it would be a nice chance to learn some new things. 

However, expect results after Monday (9/4) since, due to the Orthodox Easter (which is a national holiday in Greece), I am on vacation, away from Athens, for the next ~4 days.

**Comment by dashohoxha** [comment]
These yasks are not urgent at all. Furthermore, it is not decided yet whether you will be selected for the GSoC-2018 (things may be more clear after a week, maybe). But it is fine if you want to contribute to the project even without being selected (in your spare time, of course).

Anyway, I think that it is a good idea to list things that need to be done as issues. This may help us later to plan and organize your work. We may also try to use GitHub projects (https://github.com/Own-Mailbox/omb/projects), however I have never used them before.

Happy Easter!
